user_docs/metadata/data_dictionary/Individual.md

```diff
@@ -35,19 +35,19 @@ An Individual is a Person who is participating in a Study.
 ///
 
 ### ***geographical_region_term***
-**description** : The geographical region where the Individual is located.<br>
+**description** : The geographical region where the Individual is located. The Geographical Region is captured using a concept from the NCIT "country" class (NCIT:C25464) (e.g., Austria, Germany, Italy).<br>
```

Comment:
The Geographical Region where the Individual is located.

Reply:
will include it in the model description